extensions/binding/org.eclipse.smarthome.binding.tradfri/README.md

```diff
@@ -12,14 +12,15 @@ The thing type ids are defined according to the lighting devices defined for Zig
 |--------------------------|------------------|------------|
 | Dimmable Light           | 0x0100           | 0100       |
 | Colour Temperature Light | 0x0220           | 0220       |
-
+| Extended Colour Light    | 0x0210           | 0210       |
 
 The following matrix lists the capabilities (channels) for each of the supported lighting device types:
 
 | Thing type  | On/Off | Brightness | Color | Color Temperature |
 |-------------|:------:|:----------:|:-----:|:-----------------:|   
 |  0100       |    X   |     X      |       |                   |
 |  0220       |    X   |     X      |       |          X        |
+|  0210       |    X   |            |   X   |          X        |
```

Comment:
I am noticing that this table was already incorrect - neither 0220 nor 0210 should have an X for On/Off...

Reply:
Very good point, as there is no "onoff" channel. Should I open a new issue for this? I want to extend the documentation anyway with more examples.

Reply:
You could simply remove the column as part of this PR - or we first merge and you can come up with a new PR that enhances the documentation in general.

Reply:
I vote for option 2 - new PR when the enhancements are done.

Reply:
ok!